introduction.md

```diff
@@ -1,2 +1,5 @@
 Hi! My name is Óscar and I am a 27-years-old Engineer learning some Git!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?